bugbug/models/uplift.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, lemmatization=False):
         self.clf = xgboost.XGBClassifier(n_jobs=16)
         self.clf.set_params(predictor='cpu_predictor')
 
+    def rollback(self, change):
+        return (change['added'].startswith('approval-mozilla-') and (change['added'].endswith('+') or change['added'].endswith('-'))) or (change['removed'].startswith('approval-mozilla-') and (change['removed'].endswith('+') or change['removed'].endswith('-')))
```

Comment:
This is not really readable.
You can make it more readable by either adding temporary variables, or splitting the conditions (checking one first, then another inside it, etc.), or adding newlines between conditions.

Reply:
I think the changes i've made are wrong, and  i'll have to change the whole rollback function.

Because (approval-mozilla-) is found in "attachments" as in line 67(https://github.com/mozilla/bugbug/blob/e1bf7d6fb546b1a48e631e0854d230de669611fe/bugbug/models/uplift.py#L67) and not in bug["history"].
Right?

Reply:
Yes, that's right!

Reply:
Done.